core/src/main/scala/kafka/api/FetchRequest.scala

```diff
@@ -56,7 +58,7 @@ object FetchRequest {
         (TopicAndPartition(topic, partitionId), PartitionFetchInfo(offset, fetchSize))
       })
     })
-    FetchRequest(versionId, correlationId, clientId, replicaId, maxWait, minBytes, Map(pairs:_*))
+    FetchRequest(versionId, correlationId, clientId, replicaId, maxWait, minBytes, maxBytes, ListMap(pairs:_*))
```

Comment:
`ListMap` is not a good collection as it performs really badly unless its size is small (`get` is O(n) for example).